readme.md

```diff
@@ -450,3 +461,36 @@ body, .foo {
   [RULE_END]  
 ]
 ```
+
+#### JavaScript Function Value
```

Comment:
@kof It's a narrow path to walk, but not impossible. Here's what we're doing in the current PoC and what v3 will likely do: https://gist.github.com/philpl/1099bf4a4a28ced28dbb6b204c5fdcb3